src/rdkafka_request.c

```diff
@@ -3357,7 +3411,10 @@ rd_kafka_handle_Produce_parse(rd_kafka_broker_t *rkb,
          * request we assume that the reply only contains one topic+partition
          * and that it is the same that we requested.
          * If not the broker is buggy. */
-        rd_kafka_buf_skip_str(rkbuf);
+        if (request->rkbuf_reqhdr.ApiVersion >= 9)
```

Comment:
Revert this change as `buf_skip_str` works for flexver too now

Reply:
But we need the topic name to copy in the tags

Reply:
Sorry I meant only

```c
rd_kafka_buf_read_str(rkbuf, &TopicName);
``` 
otherwise leave it like it is but with `>= 10`

Reply:
Updated with >=10